### DISCUSSION TITLE
Option for importing content collections from remote paths

### DISCUSSION THREAD

**Discussion by divya-binil** [withastro/roadmap#663] [Proposal]
### Body

Hi, I am migrating a hugo site to astro. In Hugo, we had the option to import content using module imports in the configuration. For instance,

[module]
[[module.imports]]
  path = 'github.com/spf13/hyde'
  
It would be really good if there a similar feature available in Astro for achieving the same purpose.

**Comment by mpjijo** [comment]
Absolutely, our requirement aligns in the same manner. The availability of this feature would greatly assist us. Thank you
